benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp_ompss/Makefile

```diff
@@ -1,14 +1,8 @@
-# Makefile for manual C++ version of PSyKAl'd NEMOLite2D.
+# Makefile for manual C++ OmpSs version of PSyKAl'd NEMOLite2D.
 #
 # This Makefile picks up the compiler to use any flags from
```

Comment:
Should this now say use `../../../../../compiler_setup/ompss.sh`?

Reply:
Done